documentation/volume-tracking/docs/introduction.md

```diff
@@ -0,0 +1,13 @@
+# Volume Tracking
+
+Volume tracking plays a crucial role in the accuracy and reproducibility of DNA long read sequencing, particularly when dealing with primary and derived aliquots. In DNA sequencing, particularly with long reads, the integrity and consistency of the sample volumes are essential for ensuring that the sequencing results are reliable and representative of the original biological material.
+
+Long read sequencing technologies, such as those provided by platforms like PacBio[^1] enable the reading of extensive sequences of DNA in a single run, offering significant advantages for complex genome assemblies, structural variation analysis, and epigenetic studies. However, these technologies are highly sensitive to variations in sample preparation, making precise volume management essential from the initial extraction of DNA (primary aliquots) through the subsequent handling and processing (derived aliquots).
+
+Primary aliquots refer to the initial samples obtained directly from the DNA extraction process. These aliquots serve as the foundation for all subsequent sequencing and analysis steps. Derived aliquots, on the other hand, are portions of the primary aliquot that have been further processed, diluted, or otherwise altered to suit specific experimental needs. As each processing step introduces potential variations in volume, precise tracking of these changes is vital to maintain the integrity of the sequencing data.
```

Comment:
Primary aliquots can belong to a pool or library, not sure if that counts as part of the DNA extraction process.

Reply:
Yeah this introduction will not be part of the doc. I'm removing it :D